### PR TITLE
Add button to block artist/album/track

### PR DESCRIPTION
**Pull request by yayuyokitano**
Includes part in options to see and remove them.
Draft for now:
- [x] Implement iOS UI

### CODE REVIEW

**Comment by yayuyokitano** [comment]
Undrafted, still in need of heavy testing though since this affects pretty core functionality

File: src/_locales/en/messages.json [hunk 1]
```diff
@@ -255,6 +255,18 @@
         "message": "$1 regex/bulk edits",
         "description": "'Regex/Bulk edits' header"
     },
+    "optionsBlockedTags": {
+        "message": "Blocked tags",
+        "description": "'Blocked tags' header"
+    },
+    "optionsBlockedTagsDesc": {
+        "message": "This extension allows you to block certain artists/albums/tracks from scrobbling.",
```
Comment (inverse) [inline src/_locales/en/messages.json:263]: This copy seems inconsistent with the others on the edit page - do you think we should align them?

**Review by inverse — APPROVED** [review]
Seems legit to me

**Comment by yayuyokitano** [comment]
Significant merge

**Comment by yayuyokitano** [comment]
Redrafting, there is a race condition

**Comment by yayuyokitano** [comment]
This feels good again now, I'm undrafting.
Also this bug accidentally uncovered a different bug that has been in prod all along just not been that noticeable lol

**Pull request merged**